packages/core/test/ovm/Finder.js

```diff
@@ -0,0 +1,58 @@
+/* External Imports */
```

Comment:
Why is there a separate folder for tests for optimism? I thought we were going to use the same tests for common contracts and just blacklist certain directories that weren't OVM compatible?

Reply:
I'm guessing the point of this was to prove we can run _some_ tests in this context. Once #3180 has been merged hopefully we can move to the blacklist syntax.